pkgs/games/factorio/default.nix

```diff
@@ -186,6 +186,9 @@ let
         libXrandr
         libXi
         libGL
+        xlibs.libSM
+        xlibs.libICE
+        xlibs.libXext
```

Comment:
```suggestion
        libSM
        libICE
        libXext
```

Reply:
Yeah, that works. Thanks also for the hint regarding aliases, didn't know that!